src/boot/StylesProvider.js

```diff
@@ -36,7 +38,11 @@ class StyleProvider extends PureComponent<Props> {
   getChildContext() {
     const { theme } = this.props;
     const styles = StyleSheet.create(themeCreator(themeNameToObject[theme]));
-    return { theme, styles };
+    const navBarSizes = {
```

Comment:
You don't need to use context to pass down these properties. (they are constant!)

Reply:
@borisyankov OK, now they are in their own style in 0d86fe4, like you suggested, so they are not in `context` anymore. Thanks!